src/VSCodeExtension/src/tests/unit/control-structure.test.ts

```diff
@@ -95,4 +95,36 @@ describe("space after if rule", () => {
 
     assert.equal(formatter(code, [spaceAfterIf]), expectedCode);
   });
+
+  it("does not formats space after if statement if there is an identifier that starts with if", () => {
+    const code = "ifunctionWithParams(parameter);"
+    const expectedCode = "ifunctionWithParams(parameter);"
+
+    assert.equal(formatter(code, [spaceAfterIf]), expectedCode);
+  });
+
+  it("does not formats space after if statement if there is an identifier that ends with if", () => {
+    const code = "functionWithParamsif= 2;"
```

Comment:
Can this case (or an additional one) also check if the behavior is correct when it's a invocation? As in, `functionWithParamsif(2);`

Reply:
Thar case is already implemented :) line 99

Reply:
And line 55 https://github.com/microsoft/qsharp-compiler/pull/538/files/0c175803b781b5895195f7889433164fbef6a672#diff-a737a0ce651d7ed7689329bc1b5a97d5R55

Reply:
Ah, I see that one on 55 now! Because it wasn't changed it was skipped in the diff. Thanks for pointing it out!